x-pack/plugins/apm/server/lib/search_strategies/queries/get_query_with_params.ts

```diff
@@ -43,7 +43,7 @@ export const getQueryWithParams = ({ params, termFilters }: QueryParams) => {
     getOrElse<t.Errors, { start: number; end: number }>((errors) => {
       throw new Error(failure(errors).join('\n'));
     })
-  ) as Setup & SetupTimeRange;
+  ) as Setup & { start: number; end: number };
```

Comment:
This looks a bit odd. It looks like we are creating a fake `setup` object that actually only contains `start` and `end`.

Reply:
Sorry, I saw the comments after merging. I can do a small PR to fix this 🙏

Reply:
Thanks!